ValidationApp/src/Scenes/ParticleScene.hpp

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include <Engine/Delta.hpp>
+
+class ParticleScene : public Scene
+{
+public:
+	ParticleScene(const std::string& name)
+		: Scene(name)
+	{
+		std::shared_ptr<GameObject> particleEmitterObject = Instantiate({{20.0f, 10.0f}, 0.0f, {1.0f, 1.0f}});
+		ParticleEmitter* particleEmitter = particleEmitterObject->AddComponent<ParticleEmitter>(
+			// https://discussions.unity.com/t/2d-pixel-art-particles-tutorial/601613
```

Comment:
is deze link nog nodig? zou em misschien zo nodig ergens in de discord bewaren ipv in comments